### PR TITLE
[WIP] Team strategy templates

### PR DESCRIPTION
**Pull request by harisood**

### CODE REVIEW

**Review by cassgvp — APPROVED** [review]

**Review by cassgvp — APPROVED** [review]

**Review by cassgvp — APPROVED** [review]

**Review by cassgvp — APPROVED** [review]

File: docs/team_strategy/facilitator_notes.md
Comment (dingaaling) [inline docs/team_strategy/facilitator_notes.md:15]: I think I'd prefer using "Participant" in these resources instead of "Stakeholder". What do you think?

File: docs/team_strategy/facilitator_notes.md
Comment (dingaaling) [inline docs/team_strategy/facilitator_notes.md:23]: I think I would benefit from understanding further the value of "don't revisit" and what problem that addresses

**Review by dingaaling — CHANGES_REQUESTED** [review]
Looks great overall! I've added some comments for suggested expansion before approving the PR

File: docs/team_strategy/facilitator_notes.md
Comment (harisood) [reply docs/team_strategy/facilitator_notes.md:14]: Have made updates!

File: docs/team_strategy/facilitator_notes.md
Comment (harisood) [reply docs/team_strategy/facilitator_notes.md:15]: Yessss get me away from stakeholder!!

File: docs/team_strategy/facilitator_notes.md
Comment (harisood) [reply docs/team_strategy/facilitator_notes.md:23]: Made some edits to try and clarify

**Comment by harisood** [comment]
@dingaaling thank you for review! Check latest commit for edits based on your comments!! 

Also don't merge this yet, I'm planning on doing the rest of the sessions too before merging

**Pull request merged**
